### ISSUE TITLE
Final Report Peer Review

### ISSUE THREAD

**Issue by sukritipoddar** [PPPYL831/Breast-Cancer-Diagnosis-Prediction#12]
The project is about Breast Cancer binary classification as Benign or Malignant. The dataset used for model learning and validation is the Wisconsin Breast Cancer Malignant dataset that has 31 usable attributes and 569 samples. The team employs various classification models like Random Forest, Logistic Regression, SVC, etc. They then discuss the results. 
Strengths:
The Radar graphs are a good representation of comparison between attributes of the dataset for the target classes and mean and worse records.
The project uses methods covered in class.
They use a good number of classification models and F1 score is used in results.

Areas of Improvement:
The exploratory data analysis does not cover outlier detection. An EDA on presence of outlier detection and whether the outlier presence is justified or not should be covered due to the sensitive nature of the problem the project is investigating. 
Since the class distribution is of the about 70 % and 30% for target classes, a stratified sampling would have been better for a train test split. 
A section on model selection and why the model is chosen with interpretations on why those scores are obtained would help get more clarity and confidence in the model. 
I suggest use of regularization in models as a future work and some hyperparameter tuning based on results of the model on validation set. Since no hyperparameter tuning was done, splitting the dataset into both validation and test is rather redundant over here, as both act as validation set and also since all the sections have gold standards available.  
Heatmaps might be a better substitute for pairplots to study the correlation between features and do some feature selection accordingly. 
The report discusses the classifiers at length. Since the target audience (the class) already knows about it, it would have been better to explain the interpretation of models and comparison between scores achieved by the models.
The focus of the results should have minimum number of False Negatives for Breast Cancer detection. A table discussing the False negative scores by each model should be done and this should be a primary factor in model selection. The current model seems to give better False Positive results than False Negative.
